packages/ffmpeg/build.sh

```diff
@@ -4,10 +4,10 @@ TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
 # Please align version with `ffplay` package.
 TERMUX_PKG_VERSION="6.1.2"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_REVISION=1.1
```

Comment:
FYI, `TERMUX_PKG_REVISION` is an integer value.
It describes how many revisions there have been to a package since the last upstream package update.
E.g.
```yml
6.1.2 # initial 6.1.2 upstream release
6.1.2-1 # first packaging revision made to 6.1.2
6.1.2-2 # second packaging revision made to 6.1.2
# ...
```